src/Hamiltonians/HubbardMom1D.jl

```diff
@@ -1,18 +1,21 @@
 """
-    hubbard_dispersion(k)
-Dispersion relation for [`HubbardMom1D`](@ref). Returns `-2cos(k)`.
+    hubbard_dispersion(t,k)
+Dispersion relation for [`HubbardMom1D`](@ref). Returns `-2(Re(t) cos(k) + Im(t) sin(k))`.
```

Comment:
Maybe it is better to write this as a math formula, i.e. with double back stroke?
```suggestion
Dispersion relation for [`HubbardMom1D`](@ref). Returns ``-2(\\Re(t) \\cos(k) + \\Im(t) \\sin(k))``.
```
You'll need to check whether this works. Otherwise, if you typeset it as code, please use valid notation, i.e. using multiplication signs where needed, i.e. `Re(t) * cos(k)`.